src/blob/index/benchmarks.rs

```diff
@@ -97,7 +92,7 @@ async fn benchmark_from_records() {
         println!("Test {}...", i + 1);
         let _ = FileIndexStruct::from_records(
             Path::new(FILEPATH),
-            ioring.clone(),
+            IODriver::default(),
```

Comment:
This line is inside a loop, so it will lead to creation of multiple instances of `IODriver`. This is bad for the `Rio` case. I suggest to create single instance at the beginning of this function and pass its clones here

Reply:
Why? Default creator for async iodriver does not create `Rio`, and any default creator should be fast enough, if not const